lisa/testsuite.py

```diff
@@ -222,7 +219,9 @@ def simple_requirement(
     platform_types = search_space.create_set_space(
         supported_platform_type, unsupported_platform_type, "platform type"
     )
-
+    if unsupported_os is None:
+        if supported_os is None or Windows not in supported_os:
```

Comment:
Add a comment to explain why Windows is excluded. Because our most test cases are applied to Linux, so we exclude Windows by default.